code/game/jobs/job/engineering.dm

```diff
@@ -17,7 +17,8 @@
 			            access_heads, access_construction, access_sec_doors,
 			            access_ce, access_RC_announce, access_keycard_auth, access_tcomsat, access_minisat, access_mechanic, access_mineral_storeroom)
 	minimal_player_age = 21
-
+	exp_requirements = 600
+	exp_type = "Engineering"

```

Comment:
no magic strings please, use defines for all of these
`XP_TYPE_ENGINEERING` or something